toolbox/swiftsim/particle_splitting/generate_splitting_information.py

```diff
@@ -67,6 +67,38 @@ def load_hbt_config(config_path):
 
     return config
 
+def load_overflow_data(path_to_split_log_files):
+    '''
+    Loads particle splitting information for particles which
+    split enough times to overflow the SplitTrees dataset.
+
+    Parameters
+    ----------
+    config_path : str
```

Comment:
This should be `path_to_split_log_files`, and the description be updated.

Reply:
https://github.com/SWIFTSIM/HBTplus/pull/61/commits/046944ff50720e402bc1a9b4f2df6a83ec5ba9e4